cirq/google/experimental/noise_models.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2019 The Cirq Developers
```

Comment:
Do we expect `cirq/google/experimental` to be small enough that `noise_models` doesn't need a subdirectory?

Reply:
I guess we can keep the structure, but yeah I'm not sure how big experimental is going to grow to. I'll create the folder.